### PR TITLE
SEAR-136 No link to facility profile when no name is defined for the facility

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/secure/RapidBoard.jspa?rapidView=209&projectKey=SEAR&modal=detail&selectedIssue=SEAR-136

#### Brief feature description
- Add n/a to facility name if it is empty.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- no

### CODE REVIEW

**Comment by Nasir2712** [comment]
jenkins retest this please

File: test/javascript/search/searchList_tests.js [hunk 1]
```diff
@@ -82,7 +82,8 @@ describe('Render search list and each method', function () {
       county: null,
       phones: null,
       addresses: null,
-      type: null
+      type: null,
+      facility_source: null
```
Comment (anuroop15) [inline test/javascript/search/searchList_tests.js]: we do have a facility_source property for searchResults array on line 36. do we need this?? and any test case to check empty string value?

**Comment by Nasir2712** [comment]
@ratneshraval @anuroop15  comments addressed.

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by sunilb0575** [comment]
jenkins, retest  this please

**Comment by Nasir2712** [comment]
jenkins, retest this please
